src/renderer/src/components/interaction/InteractionTemplate.tsx

```
@@ -20,33 +20,41 @@ import {
   fetchInteractionTemplate,
   selectInteractionTemplateById,
 } from './InteractionTemplatesSlice';
+import {
+  selectInteractionStatus,
```

ESLint

[error] 'selectInteractionStatus' is defined but never used.

(@typescript-eslint/no-unused-vars)